src/test/java/edu/harvard/hul/ois/fits/junit/DpxXmlUnitTest.java

```diff
@@ -0,0 +1,119 @@
+/* 
+ * Copyright 2009 Harvard University Library
+ * 
+ * This file is part of FITS (File Information Tool Set).
+ * 
+ * FITS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * FITS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with FITS.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package edu.harvard.hul.ois.fits.junit;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.util.Scanner;
+
+import org.jdom.output.Format;
+import org.jdom.output.XMLOutputter;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import edu.harvard.hul.ois.fits.Fits;
+import edu.harvard.hul.ois.fits.FitsOutput;
+import edu.harvard.hul.ois.fits.tests.AbstractXmlUnitTest;
+
+
+public class DpxXmlUnitTest extends AbstractXmlUnitTest {
+
+	/*
+	 *  Only one Fits instance is needed to run all tests.
+	 *  This also speeds up the tests.
+	 */
+	private static Fits fits;
+
+	@BeforeClass
+	public static void beforeClass() throws Exception {
+		// Set up FITS for entire class.
+		fits = new Fits();
+	}
+
+	@AfterClass
+	public static void afterClass() {
+		fits = null;
+	}
+
+	@Test
+	public void testDpxOutput() throws Exception {
+		String inputFilename = "00266.dpx";
+    	File input = new File("testfiles/" + inputFilename);
+    	FitsOutput fitsOut = fits.examine(input);
+    	fitsOut.saveToDisk("test-generated-output/" + inputFilename + ACTUAL_OUTPUT_FILE_SUFFIX);
+
+		XMLOutputter serializer = new XMLOutputter(Format.getPrettyFormat());
```

Comment:
It looks like this file contains mixed spacing, I'm guessing copied over from another file. While I do hope to normalize all of this later (https://github.com/harvard-lts/fits/pull/310), I think it would be nice if new work did not contain mixed spacing.